docs/api.md

```diff
@@ -909,8 +909,8 @@ void vantiq.subscribe(String resource, String name, TypeOperation operation, Sub
 
 Name | Type | Required | Description
 :--: | :--: | :------:| -----------
-resource | String | Yes | The resource event to subscribe.  Must be either SystemResources.TOPICS.value(), SystemResources.SOURCES.value() or SystemResources.TYPES.value().
-name     | String | Yes | The resource name that identifies the specific resource event.  For topics, this is the topic name (e.g. '/my/topic/').  For sources, this is the source name.  For types, this is the data type name.
+resource | String | Yes | The resource event to subscribe.  Must be either SystemResources.TOPICS.value(), SystemResources.SOURCES.value(), SystemResources.SERVICES.values(), or SystemResources.TYPES.value().
```

Comment:
Should that be SystemResources.SERVICES.value() instead of SystemResources.SERVICES.values()? It looks inconsistent with the others.